inject-web3.js

```diff
@@ -0,0 +1,8 @@
+const { setWeb3 } = require('./src/global-web3');
+
+module.exports = function (web3) {
+  // [TODO] check version
```

Comment:
Isn't this done in `global-web3`?

Reply:
Ah, yes, I forgot to remove this comment.